src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -72,7 +73,7 @@ public Task<bool> RunAsync(IUpgradeContext context, IProject project, Cancellati
 
             return _runner.RunProcessAsync(new ProcessInfo
             {
-                Command = Path,
+                Command = DotNetCli,
```

Comment:
Why the change to .NET CLI?

Reply:
Because we got too long of a path with the shim that the dotnet tool installed now. I've changed it to grab just the try-convert bin directory now, which is just the managed stuff. Probably better any way now that we're going to support mac

Reply:
yeah the try-convert tool install path has become really long :)